pipeline/main.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=W0718, W1203
+
+"""Main file to run the entire ETL process"""
+
+import logging
+from extract import extract_process
+from transform import transform_process
+from load import load_process
+
+
+def run_pipeline():
+    logging.basicConfig(level=logging.INFO,
+                        format='%(asctime)s - %(levelname)s - %(message)s')
+    
+    logging.info('Starting pipeline')
+
+    try:
+        extracted_data = extract_process()
+    except Exception as e:
+        logging.error(f'Error during extraction: {e}')
+
+    try:
+        transformed_data = transform_process(extracted_data)
+    except Exception as e:
+        logging.error(f'Error during transform: {e}')
```

Comment:
For this I'd say add a return statement if there is failure, for example:
`try:
        extracted_data = extract_process()
  except Exception as e:
        logging.error(f'Error during extraction: {e}')
        return
`
That way it doesn't continue when it's not necessary